lib/drab/channel.ex

```diff
@@ -98,6 +116,10 @@ defmodule Drab.Channel do
 
           Examples:
       #{examples |> Enum.join("\n")}
+
```

Comment:
IMO this is not necessary, examples should not be too long - you see them all the time socket reloads, may be annoying ;)
Also, not everyone is using `Drab.Live` - in this case this example may be confusing.

Reply:
Do you think it could be an idea to add it to the info text conditionally when `assign_example` is non-nil?

Reply:
Nope, it is not worth to add it to those examples at all. It would be way too much - it is just a teaser, they should not replace docs and tutorial :)

Reply:
Allright, I've now removed it completely.